build/webapp-optimize.js

```diff
@@ -40,72 +40,76 @@ var HTMLOptimizer = function(options) {
 };
 
 HTMLOptimizer.prototype.process = function() {
-  var mozL10n = this.win.navigator.mozL10n;
   this.mockWinObj();
 
-  this.getAST = mozL10n.getAST.bind(mozL10n);
-
-  var ignore = this.optimizeConfig.L10N_OPTIMIZATION_BLACKLIST;
-  // If this HTML document uses l10n.js, pre-localize it --
-  //   note: a document can use l10n.js by including either l10n.js or
-  //   localization resource link elements (see /shared/js/lazy_l10n.js).
-  if ((!this.win.document.querySelector('script[src$="l10n.js"]') &&
-       !this.win.document.querySelector('link[rel="localization"]')) ||
-      ignore[this.webapp.sourceDirectoryName]) {
-    return;
-  }
-
-  // Since l10n.js was read before the document was created, we need to
-  // explicitly initialize it again via mozL10n.bootstrap, which looks for
-  // rel="localization" links in the HTML and sets up the localization context.
-  mozL10n.bootstrap(this.webapp.url, this.config);
-  this._optimize();
```

Comment:
@rickychien I realized that `this._optimize` was only called when the document passed the `if` test above, i.e. it was localization-enabled.  Is this intended?  I'll keep this the same way in the PR, but I'm wondering if it's a bug.